.github/workflows/release.yml

```diff
@@ -3,13 +3,21 @@ on:
   release:
     types: [published]
 jobs:
-  build_x86_64:
-    name: Prebuild x86_64
+  build:
+    name: Prebuild node ${{ matrix.node }} on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-13, ubuntu-22.04, ubuntu-24.04, windows-2022]
+        os: [
+          macos-13,
+          macos-14,
+          macos-15,
+          ubuntu-22.04,
```

Comment:
Well here you'll want to add ubuntu-22.04-arm no?

Reply:
Ok, I see that it is not the same version. I'll add it.